vector/src/main/java/im/vector/app/features/userdirectory/UserListViewModel.kt

```diff
@@ -241,3 +254,10 @@ class UserListViewModel @AssistedInject constructor(@Assisted initialState: User
         setState { copy(pendingSelections = selections) }
     }
 }
+
+private fun UserListViewState.hasNoIdentityServerConfigured() = matchingEmail is Fail && matchingEmail.error == IdentityServiceError.NoIdentityServerConfigured
+
+/**
+ * Wrapper class to allow identical search terms to be re-emitted
+ */
+private data class UserSearch(val searchTerm: String, val cacheBuster: Long = 0)
```

Comment:
I guess the cacheBuster param is useful behind the scene? (I do not see any `distinct()` call or similar)

Reply:
yep, it's because `SharedFlow` is distinct by default